integration_tests/src/test/kotlin/integration/DumpShardsIT.kt

```diff
@@ -0,0 +1,51 @@
+package integration
+
+import FlankCommand
+import com.google.common.truth.Truth
+import flank.common.isWindows
+import org.junit.Assume
+import org.junit.Test
+import run
+
+class DumpShardsIT {
+    private val name = this::class.java.simpleName
+
+    @Test
+    fun `dump shards - android`() {
+        val name = "$name-android"
+        val result = FlankCommand(
+            flankPath = FLANK_JAR_PATH,
+            ymlPath = "$CONFIGS_PATH/dump_shards_android.yml",
+            params = androidRunCommands + "--dump-shards"
+        ).run(
+            workingDirectory = "./",
+            testSuite = name
+        )
+
+        assertExitCode(result, 0)
+
+        val resOutput = result.output.removeUnicode()
+        Truth.assertThat(resOutput).containsMatch(findInCompare(name))
+        assertNoOutcomeSummary(resOutput)
```

Comment:
It would be great if the test verifies the content of created JSON as well

Reply:
Agree but I was thinking about creating an epic to add JSON verification for all integration tests. But on the other hand, adding JSON verification here could be a good start point for other tests. WDYT?

Reply:
Fine for me

Reply:
For clarification, I meant JSON with shards, not the one introduced on https://github.com/Flank/flank/pull/1559

Reply:
> Has it been tested on Windows ;)

Tested and works :)